src/librustdoc/html/render/cache.rs

```diff
@@ -226,18 +226,25 @@ fn get_index_type_name(clean_type: &clean::Type, accept_generic: bool) -> Option
             let path = &bounds[0].trait_;
```

Comment:
Seems like the function this is in should return `Vec<Symbol>` so that we can return the names of all the bounds?